src/components/header/Header.jsx

```diff
@@ -25,54 +29,58 @@ import useGetIOUKeys from '../../hooks/useGetIOUKeys'
 import Dropdown from 'react-dropdown';
 import 'react-dropdown/style.css';
 import { useCookies } from 'react-cookie';
-import * as a from '../../api/chain'
+import * as a from '../../api/chain';
+
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import MenuIcon from "@material-ui/icons/Menu";
+
 
 const LinkBehavior = forwardRef((props, ref) => (
   <RouterLink ref={ref} to={ROUTES.main} {...props} />
 ));
 
-const Header = ({ classes }) => {
+const Header = ({ classes, setMenuVisibility }) => {
   const [isNotifications, setIsNotifications] = useState(false);
   const [isFavorite, setIsFavorite] = useState(false);
   const dataIOUsList = useGetIOUs()
   const dataIssuers = useGetIssuers()
   const dataIOUKeys = useGetIOUKeys()
-  const dappChains = require("../../assets/dappChains.json") 
+  const dappChains = require("../../assets/dappChains.json")
   const [cookies, setCookie] = useCookies(['currChainId']);
   // const [chainId] =useContext(ChainWebContext)
   const dappStaff = require("../../assets/dappStaff.json")
 
   async function onSelect (e){
-    
+
     const { ethereum, web3 } = await a.detectEthereumProvider()
     await a.switchChain(ethereum, e.value)
     setCookie('currChainId', e.value, { path: '/' });
     window.location.reload();
- 
+
   }
   var tokens,keywords, issuers;
 
   if (dappChains[cookies.currChainId] === undefined) {
     return (
       <Typography component="h1" className={classes.title} style={{color:"red"}}>
-     Chain {cookies.currChainId} in not configured.
-  </Typography>
+        Chain {cookies.currChainId} in not configured.
+      </Typography>
     )
   }
 
-  if (dataIOUsList != undefined) { 
+  if (dataIOUsList != undefined) {
```

Comment:
```suggestion
  if (dataIOUsList !== undefined) {
```
Prefer strict comparison
The same below